### PR TITLE
Ericson support small screen

### PR DESCRIPTION
**Pull request by eric-son12**
pls check code!

### CODE REVIEW

File: src/app/app.component.scss [hunk 1]
```diff
@@ -55,7 +55,7 @@ body {
     ion-content {
         --padding-start: 2rem;
         --padding-end: 2rem;
-        width: calc(100% + 15px);
+        // width: calc(100% + 15px);
```
Comment (takYoon) [inline src/app/app.component.scss:58]: remove unused code

File: src/app/components/wallet-detail-card/wallet-detail-card.component.html [hunk 2]
```diff
@@ -194,7 +195,7 @@
           </div>
           <div *ngIf="isHomeCard">
             <ion-button class="btn-send" fill="clear" (click)="goToSendPageToken()">
-              <ion-icon class="icon-refresh" slot="start" src="assets/img/wallet-details/ic-send.svg"></ion-icon>
+              <ion-icon class="icon-refresh" slot="start" src="assets/img/wallet-details/ic-send-light.svg"></ion-icon>
```
Comment (takYoon) [inline src/app/components/wallet-detail-card/wallet-detail-card.component.html:198]: do we have dark theme for this icon ?

File: src/app/providers/profile/profile.ts [hunk 3]
```diff
@@ -2047,11 +2055,14 @@ export class ProfileProvider {
               walletClients = walletClients.concat(tokenClients);
             }
 
+            console.log('2*************WALLET' + this.wallet);
```
Comment (takYoon) [inline src/app/providers/profile/profile.ts:2058]: remove log

File: src/app/providers/profile/profile.ts [hunk 4]
```diff
@@ -2028,7 +2036,7 @@ export class ProfileProvider {
           .then(walletsData => {
             walletsData.unshift(firstWalletData);
             let walletClients = _.map(walletsData, 'walletClient');
-
+            console.log('1*************WALLET' + this.wallet);
```
Comment (takYoon) [inline src/app/providers/profile/profile.ts:2039]: remove log

File: src/app/providers/profile/profile.ts [hunk 5]
```diff
@@ -2047,11 +2055,14 @@ export class ProfileProvider {
...
             }
 
+            console.log('2*************WALLET' + this.wallet);
+
             this.addAndBindWalletClients({
               key: firstWalletData.key,
               walletClients
             })
               .then(() => {
+                console.log('*************WALLET' + this.wallet);
```
Comment (takYoon) [inline src/app/providers/profile/profile.ts:2065]: remove log

File: src/app/components/wallet-detail-card/wallet-detail-card.component.html (hunk 2 above)
Comment (eric-son12) [reply src/app/components/wallet-detail-card/wallet-detail-card.component.html:198]: dark - light theme the same icon

File: src/app/app.component.scss (hunk 1 above)
Comment (eric-son12) [reply src/app/app.component.scss:58]: add TODO: to check scroll if wrong in future

**Review by takYoon — APPROVED** [review]

**Pull request merged**